### PR TITLE
Seller info

### PR DESCRIPTION
**Pull request by diversifiedmechanic**
This is the basics of what I need until I can re-seed my db.  Same with the tests.  I don't have as much coverage as I would like but I will have to come back and add them on a little later.

### CODE REVIEW

File: src/lib.test.js [hunk 1]
```diff
@@ -4,47 +4,29 @@
...
 
   it('should handle recieving nothing in the body', async () => {
-    fetchMock.mock('/10000', {
-      body: JSON.stringify({ error: 'That ID does not exist in the database' }),
-      status: 200,
-    });
+    mFetch.notFoundFetch();
     const blankFoo = await getAdditionalItems(10000);
     expect(blankFoo.error).toEqual('That ID does not exist in the database');
   });
```
Comment (ZacharyAb) [inline src/lib.test.js]: lgtm

File: src/mFetch.mock.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+module.exports.properFetch = () => {
+  fetchMock.mock('/1', {
+    body: JSON.stringify(allButOne),
+    status: 200,
+  });
+};
+
+module.exports.resetMock = () => {
+  fetchMock.reset();
+};
```
Comment (ZacharyAb) [inline src/mFetch.mock.js]: Super cool tests, don't know enough about fetchMock to say much but excited to hear your comments on it.

File: jest.config.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 module.exports = {
   verbose: true,
   preset: '@shelf/jest-mongodb',
+  testEnvironment: 'jsdom',
```
[the remark this answers is not in the document]
Comment (diversifiedmechanic) [reply jest.config.js]: I was getting a bunch of really strange errors when I didn't have it.  I'm still getting errors but it's running.  I think you're right though and I should try to figure out why I was getting the first round of errors.

File: src/components/App.test.js [hunk 4]
```diff
@@ -3,11 +3,39 @@
...
+
+  it('Will make a call to the API when mounted', () => {
+    mFetch.properFetch(mFetch.testData);
+    const wrapper = shallow(<App itemId="1" />);
+    process.nextTick(() => {
+      wrapper.update();
+      expect(wrapper.state('allItems')).toEqual(mFetch.testData);
+    });
+  });
+
```
[the remark this answers is not in the document]
Comment (diversifiedmechanic) [reply src/components/App.test.js]: mFetch was a file I'm using to have all of my fetch-mock functions in one place so I don't have to write them a bunch of times over.

File: src/components/Seller.jsx [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+  sellerJoinDate,
+  sellerPicture,
+}) => {
+  const sellerStarRating = 4;
+
+  const SellerCont = styled.div`
+    padding: 30px 0px 24px 0px;
+    border-top: 1px solid #E1E3DF;
+    display:flex;
+    flex-dirction: row;
```
[the remark this answers is not in the document]
Comment (diversifiedmechanic) [reply src/components/Seller.jsx]: I did indeed, thanks!

File: src/components/Seller.jsx [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+  // sellerStarRating: PropTypes.number,
+  // sellerReviewCount: PropTypes.number,
+  // sellerTotalItems: PropTypes.number,
+  sellerCountry: PropTypes.string,
+  sellerTotalSales: PropTypes.number,
+  sellerPicture: PropTypes.string,
+  sellerJoinDate: PropTypes.string,
+};
+
+export default Seller;
```
[the remark this answers is not in the document]
Comment (diversifiedmechanic) [reply src/components/Seller.jsx]: I believe there is a way to do this.  I should look into how to do that.

File: jest.config.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (diversifiedmechanic) [reply jest.config.js]: I took it out and have 0 issues.  Thanks for the catch!

File: src/components/Seller.jsx [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+
+  const SellerPicture = styled.img`
+    height: 75px;
+    width: 75px;
+  `;
+
+  const SellerContainer = styled.div`
+    padding: 12px 0px 0px 12px;
+    line-height: 1.4;
+    display:flex;
```
[the remark this answers is not in the document]
Comment (diversifiedmechanic) [reply src/components/Seller.jsx]: good catch!

File: src/components/Seller.jsx [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+    flex-direction: column;
+  `;
+
+  const SellerInfoTop = styled.div`
+    padding: 6px 4px 0px 2px;
+    display:flex;
```
[the remark this answers is not in the document]
Comment (diversifiedmechanic) [reply src/components/Seller.jsx]: The down fall of copy/paste lol

File: src/components/Seller.jsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (diversifiedmechanic) [reply src/components/Seller.jsx]: Fixed!  It does look a lot better, thanks!

File: src/components/Seller.jsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (diversifiedmechanic) [reply src/components/Seller.jsx]: Fixed!

File: src/components/Seller.jsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (diversifiedmechanic) [reply src/components/Seller.jsx]: Fixed!

**Pull request merged**